elkscmd/gui/render.c

```diff
@@ -1,7 +1,7 @@
 #include <stdlib.h>
 #include "app.h"
 #include "render.h"
-#include "event.h"
+// #include "event.h"
```

Comment:
On your next PR, go ahead and take this out.

Reply:
Ok.